### PR TITLE
Run separate instance of OAV

### PR DESCRIPTION
**Pull request by sergey-shandar**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by sergey-shandar** [comment]
@anuchandy please don't merge until we check Travis-CI.

**Comment by sergey-shandar** [comment]
@vladbarosan @anuchandy New build process MODEL, PR_ONLY=false: https://travis-ci.org/Azure/azure-rest-api-specs/jobs/478461703

File: scripts/modelValidation.js [hunk 1]
```diff
@@ -3,24 +3,37 @@
...
-  // swaggersToProcess = swaggersToProcess.filter(function(item) {
-  //   return (item.match(/.*Microsoft.Logic.*2016-06-01.*/ig) !== null);
-  // });
+  let result = 0
   for (const swagger of swaggersToProcess) {
     try {
-      await oav.validateExamples(swagger, null, {consoleLogLevel: 'error', pretty: true});
-      oav.clearCache();
+      // await oav.validateExamples(swagger, null, {consoleLogLevel: 'error', pretty: true});
+      const r = exec(`node node_modules/oav/dist/cli.js validate-example ${swagger} --pretty`)
```
Comment (vladbarosan) [inline scripts/modelValidation.js]: why do want to spawn a new process ?

Comment (sergey-shandar) [reply scripts/modelValidation.js]: Because if we run in one process, we have out of memory exceptions.

**Review by vladbarosan — APPROVED** [review]

**Pull request merged**